src/components/Navbar/Navbar.jsx

```diff
@@ -144,3 +149,18 @@ const Navbar = () => {
 };
 
 export default Navbar;
+
+function useScroll(navRef) {
+  const lastScrollTop = useRef(0);
+  function handleScroll() {
+    const scrollTop = window.scrollY || document.documentElement.scrollTop;
+    navRef.current.style.top =
+      scrollTop > lastScrollTop.current ? "-80px" : "0px";
+    lastScrollTop.current = scrollTop;
+  }
+
+  useEffect(() => {
+    window.addEventListener("scroll", handleScroll);
+    return () => window.removeEventListener("scroll", handleScroll);
+  }, []);
+}
```

Comment:
If this is another function which is then imported , I would recommend to make a file with a meaningful name and move it to the utils folder

Reply:
Okay